### PR TITLE
LPS-151632 Move petra-log4j to apps/portal

### PR DESCRIPTION
**Pull request by julschong**

### CODE REVIEW

**Comment by julschong** [comment]
ci:test:relevant

**Comment by julschong** [comment]
ci:test:acceptance

File: modules/core/portal-bootstrap/system.packages.extra.bnd [hunk 1]
```diff
@@ -297,5 +296,6 @@ Provide-Capability:\
 	@lib/com.liferay.petra.sql.dsl.spi.jar,\
 	@lib/com.liferay.petra.string.jar,\
 	@lib/com.liferay.petra.url.pattern.mapper.jar,\
-	@lib/com.liferay.petra.xml.jar
+	@lib/com.liferay.petra.xml.jar,\
+	@lib/com.liferay.portal.log4j.jar
```
Comment (tinatian) [inline modules/core/portal-bootstrap/system.packages.extra.bnd:300]: @julschong 
1. You should not change petra.xml
2. Why log4j jar needed to be included? Can we remove it?

Comment (julschong) [reply modules/core/portal-bootstrap/system.packages.extra.bnd:300]: @tinatian Hi Tina,
I tried removing but error occurs when I started the server

**Comment by julschong** [comment]
@catarineaguiar Hi Catarine,
Could you please help me scan this test result?
Thanks!

**Comment by catarineaguiar** [comment]
@julschong no regressions found.

**Comment by julschong** [comment]
@tinatian 
Hi Tina,
Could you please help me review this pull?
Thank you!

File: modules/core/portal-bootstrap/system.packages.extra.bnd (hunk 1 above)
Comment (tinatian) [reply modules/core/portal-bootstrap/system.packages.extra.bnd:300]: @julschong What kind of error?

Comment (tinatian) [reply modules/core/portal-bootstrap/system.packages.extra.bnd:300]: @julschong I removed and see an error about portal-log4j-extender resolving, it is unable to find "com.liferay.portal.log4j" when it is deployed as the portal-log4j module had not been started at that moment. I will let this pull go and let's have a talk about this tomorrow.

Tina

**Comment by tinatian** [comment]
@shuyangzhou This pull is good to go.

Tina

**Comment by tinatian** [comment]
@julschong I am going to close this pull, please rebase and resend once the other pull is merged.

Thanks.
Tina

File: modules/core/portal-bootstrap/system.packages.extra.bnd (hunk 1 above)
Comment (julschong) [reply modules/core/portal-bootstrap/system.packages.extra.bnd:300]: Sorry for late response, I was rerunning this to get the error log as well.
OK, I will rebase this and we talk about it tomorrow

**Comment by tinatian** [comment]
@julschong Please rebase on master, let's see if we can move this forward first.

**Pull request closed without merging**